dashboard/app/models/census/state_cs_offering.rb

```diff
@@ -36,6 +37,13 @@ def self.construct_state_school_id(state_code, row_hash)
       School.construct_state_school_id('GA', row_hash['SYSTEM_ID'], school_id)
     when 'ID'
       School.construct_state_school_id('ID', row_hash['LeaNumber'], row_hash['SchoolNumber'])
+    when 'NC'
+      # school code in the spreadsheet from North Carolina is prefixed with the district code
+      # but our schools data imported from NCES does not
+      district_code = row_hash['NC LEA Code']
+      school_code = row_hash['NC School Code']
+      school_code.slice!(district_code)
```

Comment:
Maybe this behaviour of `slice!` is well-known, but if you want to save someone like me a few minutes of googling, you could add a comment along the lines of "Remove the district_code from the school_code."

Reply:
Done.